classes/class-wc-connect-logger.php

```diff
@@ -2,16 +2,21 @@
 
 if ( ! class_exists( 'WC_Connect_Logger' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_DEBUG_LOGGING_ENABLED_OPTION', 'wcc_debug_logging_enabled' );
+
 	class WC_Connect_Logger {
 
 		/**
 		 * @var WC_Logger
 		 */
 		private $logger;
 
+		private $is_logging_enabled = false;
+
 		public function __construct( WC_Logger $logger ) {
 
 			$this->logger = $logger;
+			$this->is_logging_enabled = ( 1 == get_option( WOOCOMMERCE_CONNECT_DEBUG_LOGGING_ENABLED_OPTION, 0 ) );
```

Comment:
Why not just `$this->is_logging_enabled = (bool) get_option( WOOCOMMERCE_CONNECT_DEBUG_LOGGING_ENABLED_OPTION );` ?